LinkitOne/bluetooth.ino

```diff
@@ -0,0 +1,64 @@
+""" This program scans for Bluetooth devices in the vicinity of LinkIt One Microcontroller. It scans once in every three seconds for such devices""""
+#include <LBT.h>
+#include <LBTClient.h>
+#include <SPI.h>
+#include <LSD.h>
+#include <Time.h>
+""" Declaring a file to be stored on the sd card """
+LFile mbt;
+
+void setup() {
```

Comment:
Please indent your code. Follow Arduino's default coding style and indent your code.